PsfRuntime/Config.cpp

```diff
@@ -299,6 +299,13 @@ void load_json()
                 }
             }
         }
+        if (!file)
+        {
+            MessageBoxEx(NULL, L"'Config.json' was not found. We recommend you to add it under the package root folder and re-install the application.", L"MSIX Packet Support Framework", MB_OK | MB_ICONERROR, 0);

```

Comment:
Same thing here.  Once the user clicks out of the message box PSF execution will continue and PSF will crash.